Remind_Device/Remind_Device.ino

```diff
@@ -53,6 +64,7 @@ void setup() {
 
   // clearing the LCD and putting the cursor at 0,0
   lcd.clear();
+  lcd.setCursor(0,0);
```

Comment:
lcd.clear() automatically puts the cursor on (0,0). This line is redundant. I'm currently working on the code as well, so I'll edit it.